hunting-buddy.lic

```diff
@@ -49,16 +49,32 @@ class HuntingBuddy
       end
 
       args = info['args']
+      before_actions = info['before']
+      after_actions = info['after']
       duration = info[:duration]
       stop_on_skills = info['stop_on']
 
+      #Execute scripts to run before the hunt
```

Comment:
I recommend extracting this into a helper method. That keeps this main method more about flow, abstracting the details out to the helper.